diff -up dvdrip-0.98.9/lib/Video/DVDRip/Term/Main.pm~ dvdrip-0.98.9/lib/Video/DVDRip/Term/Main.pm
--- dvdrip-0.98.9/lib/Video/DVDRip/Term/Main.pm~	2006-08-16 21:34:38.000000000 +0200
+++ dvdrip-0.98.9/lib/Video/DVDRip/Term/Main.pm	2008-12-16 02:16:15.000000000 +0100
@@ -20,7 +20,6 @@ use strict;
 use Video::DVDRip::Project;
 use Video::DVDRip::Logger;
 use Video::DVDRip::Term::Progress;
-use Video::DVDRip::GUI::Pipe;
 use Video::DVDRip::Term::ExitTask;
 
 sub filename            { shift->{filename}                         }
